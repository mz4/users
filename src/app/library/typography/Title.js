import classnames from 'classnames';
import styles from './Title.module.scss';

export const Title = ({ text, customClassName = '' }) => {
  return (
    <div className={classnames(styles.pageTitle, customClassName)}>
      <span>{text}</span>
    </div>
  );
};
