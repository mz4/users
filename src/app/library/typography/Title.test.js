import React from 'react';
import renderer from 'react-test-renderer';
import { Title } from './Title';

describe('Render title component', () => {
  it('Create a snapshot', () => {
    const title = <Title />;
    const component = renderer.create(title);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
