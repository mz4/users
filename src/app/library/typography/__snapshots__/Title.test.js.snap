// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render title component Create a snapshot 1`] = `
<div
  className="pageTitle"
>
  <span />
</div>
`;
